src/Microsoft.Owin.Security.Interop/Microsoft.Owin.Security.Interop.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>A compatibility layer for sharing authentication tickets between Microsoft.Owin.Security and Microsoft.AspNetCore.Authentication.</Description>
-    <TargetFramework>net451</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
Isn't the bar now a bit high for a package meant to be consumed by OWIN/Katana apps that are often hosted on servers that don't have the latest (and greatest) .NET Framework bits?